aspnetcore/blazor/fundamentals/routing.md

```diff
@@ -1902,6 +1920,116 @@ In the following `App` component example:
 > [!NOTE]
 > Not throwing if the cancellation token in <xref:Microsoft.AspNetCore.Components.Routing.NavigationContext> is canceled can result in unintended behavior, such as rendering a component from a previous navigation.
 
+## Handle location changes in a component
```

Comment:
Should we call this section something like "Handle/prevent location changes"? I ask because the ability to prevent navigation events was the biggest motivator for this feature.

Reply:
"Handle" just means "handle" in an arbitrary way. However, making a popular feature stand out is a good idea.

🤔 ...

*Handle (prevent) location changes*

Does that work?